src/kernel/chainparams.cpp

```diff
@@ -495,12 +495,19 @@ class CRegTestParams : public CChainParams
         };
 
         m_assumeutxo_data = {
-            {
+            {   // For use by unit tests
```

Comment:
I didn't do that because there are already multiple files that use it (`validation_chainstatemanager_tests`, `validation_chainstate_tests`). Seems less likely that the same would happen for fuzz tests.